features/roomdetails/impl/src/main/kotlin/io/element/android/features/roomdetails/impl/RoomDetailsPresenter.kt

```diff
@@ -108,6 +123,15 @@ class RoomDetailsPresenter @Inject constructor(
         return canInvite
     }
 
+    @Composable
+    private fun getCanSendState(membersState: MatrixRoomMembersState, type: StateEventType): State<Boolean> {
```

Comment:
Perhaps this was to be named `getCanEditState` ?
(same goes for the `val` in the body)